fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -0,0 +1,89 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.system;
+
+import com.starrocks.alter.DecommissionBackendJob;
+import com.starrocks.common.FeMetaVersion;
+import com.starrocks.common.io.Text;
+import com.starrocks.server.GlobalStateMgr;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+public class ComputeNode extends Backend {
+
+    private static final Logger LOG = LogManager.getLogger(ComputeNode.class);
+
+    public ComputeNode() {
+    }
+
+    public ComputeNode(long id, String host, int heartbeatPort) {
+        super(id, host, heartbeatPort);
+    }
+
+    @Override
+    public void write(DataOutput out) throws IOException {
+        out.writeLong(getId());
```

Comment:
Manual serialization is deprecated, should be replaced with JSON serialization.

Reply:
it has change to JSON serialization.